dctools/CMakeLists.txt

```diff
@@ -0,0 +1,128 @@
+# Data Center Tools
+
+if(THEROCK_ENABLE_RDC)
+  # RDC builds with statically linked gRPC; tools (protoc, grpc_cpp_plugin) come from the bundled
+  # therock-grpc stage dir to keep symbol visibility and dependencies consistent with RFC0007.
+  if(NOT TARGET ${THEROCK_BUNDLED_GRPC})
+    message(FATAL_ERROR "RDC requires bundled gRPC (${THEROCK_BUNDLED_GRPC}); enable THEROCK_BUNDLE_SYSDEPS=ON to build gRPC and its dependencies.")
+  endif()
+  get_target_property(_therock_grpc_stage ${THEROCK_BUNDLED_GRPC} THEROCK_STAGE_DIR)
+  if(NOT _therock_grpc_stage)
+    message(FATAL_ERROR "Bundled gRPC target ${THEROCK_BUNDLED_GRPC} is missing THEROCK_STAGE_DIR")
+  endif()
+  set(_therock_grpc_root "${_therock_grpc_stage}/lib/rocm_sysdeps")
+  set(_rdc_grpc_cmake_dir "${_therock_grpc_root}/lib/cmake/grpc")
+  set(_rdc_protobuf_cmake_dir "${_therock_grpc_root}/lib/cmake/protobuf")
+
+  set(_rdc_runtime_deps
```

Comment:
We hardly ever do this unless if there is conditional logic. Just inline into RUNTIME_DEPS.